src/networkcreator.js

```diff
@@ -1055,6 +1057,60 @@ class NetworkCreator extends EventEmitter {
     return needToCreateGeth(this.config) ? gethService : undefined
   }
 
+  generateGethFaucet (volumes) {
+    let faucetService = {
+      image: 'vergauwennico/livepeer-testnet:faucet',
```

Comment:
Now that we're about to integrate these services, can we move the repos into the Livepeer org and push the images into Livepeer DockerHub to make future maintenance easier?

Reply:
Sounds good ! Do you want to put it into a more general github repo such as `testnet-services` or just their own separate repos?

Reply:
👍 for putting both services in a single repo as long as there aren't any GOPATH issues (if both services are using Go mod then I don't think this should be a problem since there won't be any reliance on GOPATH)

Reply:
Dockerhub: https://cloud.docker.com/u/livepeer/repository/docker/livepeer/testnet-services

Github: https://github.com/livepeer/testnet-services

& addressed in 06d3732

Reply:
> set up building images from github , wdyt ? 

Yeah would be great to set up CI for that repo to automatically push updates images to DockerHub if a PR is merged.